src/unitTests/ApiTests.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace unitTests
+{
+    public class ApiTests: IClassFixture<WebApplicationFactory<minapi.Startup>>
+    {
+        private readonly WebApplicationFactory<minapi.Startup> _factory;
+
+        public ApiTests(WebApplicationFactory<minapi.Startup> factory)
+        {
+            _factory = factory;
+            System.Environment.SetEnvironmentVariable("TEST", "Hello World!");
+        }
+
+        [Fact]
+        public void Successful()
+        {
+            Assert.True(true);
```

Comment:
![image](https://user-images.githubusercontent.com/17045624/78501044-b6776600-7762-11ea-9628-1a1bc95990fa.png)

Reply:
helped to setup unit tests, in vscode, may be removed now